### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by bigspider**
Added some additonal documentation the main README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,4 +39,73 @@ We always have an opportunity to make right our mistake and refund the customer
...
+## Build production instance
+
+```
+npm run build              # build PISA
+npm run build-client       # build the client library
+```
+
+# Hiring Pisa
+
+**Note:** Pisa is alpha stage software and is not yet available on mainnet. Do not use with real funds.
```
Comment (yahgwai) [inline README.md]: Maybe also write that all specifications and APIs are subject to change at any time

File: client/README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 A thin client for the PISA API.
 
+A live version on Ropsten can be accessed via https://alpha.pisa.watch; the PISA contract is deployed at the address 0xA02C7260c0020343040A504Ef24252c120be60b9.
```
[the remark this answers is not in the document]
Comment (bigspider) [reply client/README.md]: Right, removed.

File: README.md [hunk 3]
```diff
@@ -39,4 +39,73 @@ We always have an opportunity to make right our mistake and refund the customer
...
+    "responderKey": <private key of the wallet that the Pisa will use to send transactions>,
+    "receiptKey": <private key of the wallet that Pisa will use to sign the appointments>
+}
+```
+
+Run `npm run start -- --help` to see all the available settings. The same settings can also be provided via the command line. Settings provided via the command line override the ones in `config.json`.
+
+## Run a Pisa Tower
+
+Make sure that the relevant ports are open. You can run Pisa with the following command:
```
[the remark this answers is not in the document]
Comment (bigspider) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (bigspider) [reply README.md]: Done.

**Pull request merged**